### ISSUE TITLE
Add different commissions rates based on user's roles or seller choices

### ISSUE THREAD

**Issue by fxbenard** [easydigitaldownloads/EDD-Commissions#132]
It will be a nice feature to be able to assign different rates to users based on our own needs.

E.g. I give a commission to 2 users but based on their role (writer or reviewer) they don't get the same commissions.

E.g. On a specific product I want to give user2 a commission of 50% and user8 only 5%.

**Comment by pippinsplugins** [comment]
@fxbenard Is there a reason you don't just set the default rate on the user's profile page?

**Comment by fxbenard** [comment]
Yes, they not always get the same rate. Sometimes they're writers, 
sometimes they're reviewers so different rates.

Le 08/04/2016 18:14, Pippin Williamson a écrit :
>
> @fxbenard <https://github.com/fxbenard> Is there a reason you don't 
> just set the default rate on the user's profile page?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/easydigitaldownloads/EDD-Commissions/issues/132#issuecomment-207497825>
>

-- 
signature FX Bénard | FxB
Website : https://fxbenard.com
Twitter : @fxbenard <https://twitter.com/fxbenard>
Google+ : Fx Bénard <https://plus.google.com/115184248259085010066/posts>

**Comment by davidsanders** [comment]
This is a feature my client is requesting. Any chance we could have this bumped up in the queue? Seems relatively easy to do as the override mechanism is already in place at a more granular level for product.

**Comment by davidsanders** [comment]
@pippinsplugins This is a feature my client is requesting, and the last feature to check off for us to dive into the EDD eco-system for their marketplace. Any chance we could have this bumped up in the queue? It would seem relatively easy to do as the override mechanism is already in place at a more granular product level.

**Comment by pippinsplugins** [comment]
@davidsanders We can maybe look at it further after the 3.3 release is completed.

In the mean time, are you familiar with the per-user rates that can be set?

**Comment by davidsanders** [comment]
@pippinsplugins yes we're aware of the product level override. However, at scale, this will be an arduous manual process to update dozens of products, individually, as it correlates to a vendor's membership level (role). Is the commission rate stored at the product meta level such that it could be set via an externalized script?

**Comment by pippinsplugins** [comment]
3.3 should be completed within next couple of weeks.

**Comment by davidsherlock** [comment]
@davidsanders, I am going to build a small plugin for this soon. I believe this request should third party and not core Commissions.
